blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,413 @@
+.v2-footer-wrapper {
+    background: var(--c-primary-black);
+  }
+  
+  .v2-footer {
+    color: var(--c-secondary-silver);
+    padding: 60px 32px 10px;
+    font-size: 13px;
+    margin: auto;
+    max-width: 1376px;
+  }
+  
+  .v2-footer ul {
+    list-style: none;
+    padding: 0;
+  }
+  
+  .v2-footer .v2-footer-social-media-section {
+    margin-top: 21px;
+    display: flex;
+    gap: 20px;
+    flex-wrap: wrap;
+  }
+  
+  .v2-footer .v2-footer-social-media {
+    border-radius: 17.5px;
+    color: var(--c-tertiary-cool-gray);
+    text-decoration: none;
+    display: flex;
+    width: 30px;
+    height: 30px;
+    align-items: center;
+    justify-content: center;
+    font-size: 20px;
+  }
+  
+  .v2-footer .v2-footer-social-media:hover {
+    color: var(--c-tertiary-cool-white);
+  }
+  
+  .v2-footer .v2-footer-social-media::before {
+    width: 17px;
+  }
+
+  .icon-fb,
+  .icon-twitter,
+  .icon-yt,
+  .icon-instagram,
+  .icon-linkedin {
+    display: flex;
+  }
+
+  
+  .v2-footer .v2-footer__title {
+    font-size: 14px;
+    font-weight: 400;
+    margin-bottom: 5px;
+    line-height: 22px;
+  }
+  
+  .v2-footer ul li a,
+  .v2-footer .v2-footer-bottom-section li a {
+    color: var(--c-secondary-silver);
+  }
+  
+  .v2-footer ul li a:hover {
+    color: var(--c-secondary-white);
```

Comment:
This variable isn't defined, so i can't see a hover state

Reply:
Fixed